src/frontend/components/SubscribeScheduledVideo/index.spec.tsx

```diff
@@ -0,0 +1,429 @@
+import {
+  act,
+  fireEvent,
+  render,
+  waitFor,
+  screen,
+} from '@testing-library/react';
+import { Grommet } from 'grommet';
+import { videoMockFactory } from '../../utils/tests/factories';
```

Comment:
Can you use absolute imports in place of ascending relative ones in this file please ?